src/prisma/schema.prisma

```diff
@@ -3471,19 +3474,21 @@ model ScoutProjectMember {
 }
 
 model ScoutProjectContract {
-  id           String               @id @default(uuid()) @db.Uuid
-  createdAt    DateTime             @default(now())
-  createdBy    String               @db.Uuid
-  deployedAt   DateTime
-  projectId    String               @db.Uuid
-  project      ScoutProject         @relation(fields: [projectId], references: [id], onDelete: Restrict)
-  address      String
-  chainId      Int
-  deployerId   String               @db.Uuid
-  deployer     ScoutProjectDeployer @relation(fields: [deployerId], references: [id], onDelete: Restrict)
-  deployTxHash String
-  blockNumber  Int
-  deletedAt    DateTime?
+  id                                 String                               @id @default(uuid()) @db.Uuid
+  createdAt                          DateTime                             @default(now())
+  createdBy                          String                               @db.Uuid
+  deployedAt                         DateTime
+  projectId                          String                               @db.Uuid
+  project                            ScoutProject                         @relation(fields: [projectId], references: [id], onDelete: Restrict)
+  address                            String
+  chainId                            Int
+  deployerId                         String                               @db.Uuid
+  deployer                           ScoutProjectDeployer                 @relation(fields: [deployerId], references: [id], onDelete: Restrict)
+  deployTxHash                       String
+  blockNumber                        BigInt
+  deletedAt                          DateTime?
+  transactions                       ScoutProjectContractTransaction[]
+  ScoutProjectContractTransactioData ScoutProjectContractTransactioData[]
```

Comment:
small typo should be `ScoutProjectContractTransactionData` with n after transaction.